internal/handshake/xor_nonce_aead.go

```diff
@@ -0,0 +1,13 @@
+//go:build !boringcrypto
```

Comment:
I would just rename this file to _noboring suffix

Reply:
fixed